### PR TITLE
Adds hugging

### PR DESCRIPTION
**Pull request by Mirino97**
## About the PR 

HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS HUGS 
[At the request of @DogZeroX . Also, you can hug yourself. I call that a feature, but it should be easy to remove that with an if check.]

**Screenshots**
image

https://user-images.githubusercontent.com/52474532/163652605-a03c808a-37a8-4c64-95e5-4c31b6dc79e2.mp4

**Changelog**

:cl:
- add: Nanotrasen has spent billions of credits in morale research, and the newest morale-increasing technology is here: Hugging!

### CODE REVIEW

File: Resources/Prototypes/Entities/Mobs/Player/dwarf.yml [hunk 1]
```diff
@@ -5,6 +5,11 @@
   id: MobDwarf
   description: A miserable pile of secrets.
   components:
+    - type: InteractionPopup
+      successChance: 0.9
```
Comment (moonheart08) [inline Resources/Prototypes/Entities/Mobs/Player/dwarf.yml:9]: should honestly be higher.

Comment (Mirino97) [reply Resources/Prototypes/Entities/Mobs/Player/dwarf.yml:9]: I wanted a funny "haha you failed a hug and smacked the person on their head". Upping it to 1.0 would make it 100% success rate, but I can make that change if you want :D

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Audio needs a license.

File: Resources/Audio/Effects/licenses.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+hugg.ogg taken from /tg/station at https://github.com/tgstation/tgstation/blob/master/sound/weapons/thudswoosh.ogg - Under license: CC BY-SA 3.0
```
Comment (metalgearsloth) [inline Resources/Audio/Effects/licenses.txt:1]:
```suggestion
hug.ogg taken from /tg/station at https://github.com/tgstation/tgstation/blob/master/sound/weapons/thudswoosh.ogg - Under license: CC BY-SA 3.0
```

**Comment by 20kdc** [comment]
Where's hugging-failure-generic?

**Comment by Mirino97** [comment]
> Where's hugging-failure-generic?

Removed cuz failure chance was removed by @moonheart08 's request.
I'm assuming having an unused failure-generic is not necessary,

File: Resources/Prototypes/Entities/Mobs/Player/slime.yml [hunk 3]
```diff
@@ -3,6 +3,11 @@
   parent: MobSlimePersonBase
   id: MobSlimePerson
   components:
+    - type: InteractionPopup
+      successChance: 0.9
```
Comment (20kdc) [inline Resources/Prototypes/Entities/Mobs/Player/slime.yml:7]: ???????

Comment (Mirino97) [reply Resources/Prototypes/Entities/Mobs/Player/slime.yml:7]: Nice catch.  #7598

**Comment by Mirino97** [comment]
Removed as moon heart asked the success chance to be 100%

Enviado do Yahoo Mail no Android 
 
  Em sáb., 16 16e abr. 16e 2022 às 5:50, ***@***.***> escreveu:   


Where's hugging-failure-generic?

**Comment by asperger-sind** [comment]
> Removed as moon heart asked the success chance to be 100% Enviado do Yahoo Mail no Android Em sáb., 16 16e abr. 16e 2022 às 5:50, ***@***.***> escreveu: Where's hugging-failure-generic? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***>

what.

**Pull request merged**